login/src/main/resources/templates/web/accounts/new_activation_email.html

```diff
@@ -3,7 +3,27 @@
       xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
       layout:decorator="layouts/main"
       th:with="pivotal=${@environment.getProperty('login.brand') == 'pivotal'},isUaa=${T(org.cloudfoundry.identity.uaa.zone.IdentityZoneHolder).isUaa()}">
+<head>
+    <th:block layout:include="pw_validation :: deps"></th:block>
+    <th:block layout:include="pw_validation :: head"></th:block>
+</head>
 <div class="island-landscape" layout:fragment="page-content">
+    <script th:inline="javascript">
+    /*<![CDATA[*/
```

Comment:
I've never seen this needed when dumping into a script tag. I'm guessing this is something strange with this java templating?